newjersey/navigator.business.nj.gov
# Create industry specific starter kits revisions

## Description

We wanted to largely do 2 things
We wanted to have Steps be represented as Step when there was only one step with regards to the title for the Road Map Section

We also wanted all essential questions to be answered to give us the maximum number of roadmaps events for SEO

![image](https://github.com/user-attachments/assets/750a71c1-3e40-4e05-baa3-6daac0334555)




### Ticket



This pull request resolves [#187792208](https://www.pivotaltracker.com/story/show/187792208).

### Approach



### Steps to Test

Run Local and Go here: http://localhost:3000/starter-kits/domestic-employer, see that at the bottom it says Step instead of Steps for the roadmap title (because it only has 1 step)

http://localhost:3000/starter-kits/ go here and see that it says steps

We also wanted to make sure that essential questions are being answered in the affirmative to see as many roadmap steps as possible. 

Go to any of the urls for starter kits with essential questions, 

http://localhost:3000/starter-kits/cannabis or http://localhost:3000/starter-kits/certified-public-accountant

and see that the roadmaps contains extra events due to essential questions like Apply for Your Annual Cannabis License or Apply for a Certified Public Accountant License

A thing I added is the idea that we should have a test that fails if we add an essential question that isn't accounted for in the logic of the starter-kits. 

I've added this test and have a second commit with code that you can uncomment to verify that it does indeed fail if an additional essential question is added (this will be removed prior to rebasing into main)



### Notes



## Code author checklist
